### DISCUSSION TITLE
Is there a way to create a set of example of the same api endpoint

### DISCUSSION THREAD

**Discussion by cfrapper** [apideck-libraries/portman#345] [Q&A]
Hello, 
first of all thank you for your application, it's quite wonderful. 
I was wondering if there was a way with portman to dynamically create example of API endpoint with different request body and then export them to a postman collection ? 
Sincerely, 
Colin

**Comment by thim81** [comment]
Hi @cfrapper 

If you include "example" values in your OpenAPI specification include full "examples", they will be use as request body in the generated Postman collection.

Or are you looking for another case?

**Comment by cfrapper** [comment]
Hey @thim81, thank you for the response, yes indeed, this is what I use. But let's say I have two examples of a request, it will only create a single request with the first example, what I was looking for is that if I have two examples it will create two requests with the corresponding example

**Reply by thim81** [reply]
hi @cfrapper 

We have a feature request open for this: 
https://github.com/apideck-libraries/portman/issues/191
And there has been another discussion on this: https://github.com/apideck-libraries/portman/discussions/298 

The idea is that Portman would generate a variation for each request body example (except for the 1st one since that would remain the "main" request).

The biggest question is how to configure and manage it in an easy, simple manner?
In the discussion, there is a proposal, feel free to give your input on this.
